nbsphinx_link/__init__.py

```diff
@@ -11,27 +11,155 @@
     "path": "relative/path/to/notebook"
 }
 
+
+Optionally the "extra-media" key can be added, if your notebook includes
+any media, i.e. images. The value needs to be an array of strings,
+which are paths to the media files or directories.
+
 Further keys might be added in the future.
 """
 
-
 import json
 import os
+import shutil
 
 from docutils import io, nodes, utils
 from docutils.utils.error_reporting import SafeString, ErrorString
+import docutils  # noqa: F401
```

Comment:
?

Reply:
The ``# noqa: F401`` if for ``flake8`` not to complain about the unused import, since ``docutils`` is only used in the docsting for the type of ``document``.